democracy/views/hearing.py

```diff
@@ -62,23 +44,40 @@ def get_sections(self, hearing):
         serializer.bind('sections', self)  # this is needed to get context in the serializer
         return serializer.to_representation(queryset)
 
+    def get_main_image(self, hearing):
```

Comment:
Yeah good points about performance, wasn't really aiming for performance, but especially this one was way too naive. However I think this solution of yours isn't quite there, because I think we should be prepared for multiple introduction images, in which case it would possible work incorrectly.